tests/sqlparser_mssql.rs

```diff
@@ -2480,3 +2492,15 @@ fn parse_mssql_grant() {
 fn parse_mssql_deny() {
     ms().verified_stmt("DENY SELECT ON my_table TO public, db_admin");
 }
+
+#[test]
```

Comment:
Could we have a similar test or move this over to common.rs? thinking since this flag is being added to the parser itself, it feels a bit like it could be lost if only tested here within mssql